meson.build

```diff
@@ -0,0 +1,55 @@
+project('xtensor', 'cpp',
+        version:'0.21.5',
+        license:'BSD-3-Clause',
+        default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+xtl_dep = dependency('xtl',
+    version:['>=0.6.12', '<1.0.0'],
+    fallback : ['xtl', 'xtl_dep'],
+    required: false)
+
+json_dep = dependency('nlohmann_json',
+    version:'>=3.1.1',
+    fallback : ['nlohmann_json', 'nlohmann_json_dep'],
```

Comment:
Fallback should not be necessary if the wrapdb is used, because of https://github.com/mesonbuild/nlohmann_json/blob/15f5c53d5fd354469fb4f801da1030ba1106161c/upstream.wrap#L9-L10